Examples/MultiModal/Mixed_Image__And_Tabular_Reading/.ipynb_checkpoints/CompositeReader-checkpoint.ipynb

```diff
@@ -0,0 +1,429 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Composite Data Readers Tutorial**\n",
```

Comment:
Remove the checkpoint file please. Also, I left a lot of comments with the checkpoint file that needs to be incorporated into the actual ipynb file.